loleaflet/src/layer/tile/CanvasTileLayer.js

```diff
@@ -352,181 +399,29 @@ L.TileSectionManager = L.Class.extend({
 	_updateWithRAF: function () {
 		// update-loop with requestAnimationFrame
 		this._canvasRAF = L.Util.requestAnimFrame(this._updateWithRAF, this, false /* immediate */);
-		this.update();
+		this._sectionContainer.requestReDraw();
 	},
 
 	update: function () {
-
```

Comment:
Are the below checks done somewhere before attempting a full redraw ? These checks help us to avoid unnecessary redraws etc when these are done via RAF loop which is good w.r.t. cpu/battery.

Reply:
All things were checked inside update function. Now appropriate callbacks are called when an event occurs. So, we don't need them now.

I also saw that loop may need some checks but while moving, there will be changes most probably.

Thanks for the comments:)